### ISSUE TITLE
Opening VS Code causes multiple windows with outdated versions of files I've already saved to open

### ISSUE THREAD

**Issue by Valkinsenn** [microsoft/vscode#193238]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.82.2
- OS Version: Windows 10, Version 22H2 (OS Build 19045.3448)

Steps to Reproduce:

1. Open VS Code while it isn't open.

(OR)

1. Close down all the windows opened *by* VS Code (whether I do or don't save the files it opens is irrelevant).
2. Shut down VS Code.
3. Reopen VS Code.

To elaborate, I thought this was an issue with VS Code restoring windows that were previously closed down, only I turned that feature off and it still does it whenever I close down and reopen VS Code anyway. Further exacerbating the issue is that fact that it seems to forget whether or not I actually saved said files before closing because, even if I open a project folder that I had open previously, and saved every single file I was working on, it'll still act as if I hadn't saved them. Even more bizarrely, it'll tell me I have a version mismatch and ask me to compare the supposedly "newer" file to the "old" one I'm trying to save it over. I've tried everything from turning various features on or off, like the aforementioned window restore feature, to even reinstalling VS Code _itself,_ with nothing really resolving this issue, leading me to believe it's a bug of some kind.

Needless to say, this is VERY not helpful, especially seeing as how I use VS Code exclusively.

![Mismatch Screenshot](https://github.com/microsoft/vscode/assets/49046029/2fea731f-aca3-4408-959a-ae7e8903520f)

**Comment by bpasero** [comment]
This does not sound like an issue with VS Code but more with maybe something going bad when saving files on your machine, which seems to fail.

If you have clear steps to reproduce from scratch using our insiders version (https://code.visualstudio.com/insiders/) that I am able to reproduce, then I can understand if there is something going on that we need to address in VS Code.

**Comment by Valkinsenn** [comment]
Okay, so, I wasn't able to recreate the problem in the Insiders version of VS Code, so I was basically forced to take the nuclear option to resolve this problem in regular VS Code. Basically, I uninstalled Code, deleted its AppData folder entirely, then reinstalled, and it _seems_ to have fixed the problem. At least for now. Thankfully, somehow, my extensions were still installed, so I don't have to go back and reinstall them all again. If anything, I'm going to be keeping a closer eye on how many extensions I have installed and take a more active role in trimming the fat on that front.

Although, I _think_ — and this is just speculation at this point — what happened was that I was trying to open Markdown files in another working folder while I was working on a Svelte project in a completely different folder, and somehow, some way, it ended up messing some critical part of Code's save / load functionality. Can't be 100% sure, and I know this isn't helpful, but I think that's what happened. Either way, VS Code's back to working as normal (for now), so I think this issue is basically resolved at this point.
